localization/english/wc_buildings_l_english.yml

```diff
@@ -269,3 +275,632 @@
+ building_lightning_vein_mine_01_desc:1 "$building_rammelsberg_mines_01_desc$"
+ building_lightning_vein_mine_02:1 "Lightning Vein Mine"
+ building_lightning_vein_mine_02_desc:1 "$building_rammelsberg_mines_02_desc$"
+ building_lightning_vein_mine_03:1 "Large Lightning Vein Mine"
+ building_lightning_vein_mine_03_desc:1 "$building_rammelsberg_mines_03_desc$"
+ building_lightning_vein_mine_04:1 "Lightning Vein Mining Complex"
+ building_lightning_vein_mine_04_desc:1 "$building_rammelsberg_mines_04_desc$"
+ building_type_bleak_hills_mine_01:0 "Bleak Hills Mine"
+ building_type_bleak_hills_mine_01_desc:1 ""
+ building_bleak_hills_mine_01:1 "Bleak Hills Mining Settlement"
+ building_bleak_hills_mine_01_desc:1 "$building_rammelsberg_mines_01_desc$"
+ building_bleak_hills_mine_02:1 "Bleak Hills Mine"
+ building_bleak_hills_mine_02_desc:1 "$building_rammelsberg_mines_02_desc$"
+ building_bleak_hills_mine_03:1 "Large Bleak Hills Mine"
+ building_bleak_hills_mine_03_desc:1 "$building_rammelsberg_mines_03_desc$"
+ building_bleak_hills_mine_04:1 "Bleak Hills Mining Complex"
+ building_bleak_hills_mine_04_desc:1 "$building_rammelsberg_mines_04_desc$"
+ building_type_winterchill_mine_01:0 "Winterchill Mine"
+ building_type_winterchill_mine_01_desc:1 ""
+ building_winterchill_mine_01:1 "Winterchill Mining Settlement"
+ building_winterchill_mine_01_desc:1 "$building_rammelsberg_mines_01_desc$"
+ building_winterchill_mine_02:1 "Winterchill Mine"
+ building_winterchill_mine_02_desc:1 "$building_rammelsberg_mines_02_desc$"
+ building_winterchill_mine_03:1 "Large Winterchill Mine"
+ building_winterchill_mine_03_desc:1 "$building_rammelsberg_mines_03_desc$"
+ building_winterchill_mine_04:1 "Winterchill Mining Complex"
+ building_winterchill_mine_04_desc:1 "$building_rammelsberg_mines_04_desc$"
+ building_type_redrock_mines_01:0 "Redrock Mines"
+ building_type_redrock_mines_01_desc:1 ""
+ building_redrock_mines_01:1 "Redrock Mining Settlement"
+ building_redrock_mines_01_desc:1 "$building_rammelsberg_mines_01_desc$"
+ building_redrock_mines_02:1 "Redrock Mines"
+ building_redrock_mines_02_desc:1 "$building_rammelsberg_mines_02_desc$"
+ building_redrock_mines_03:1 "Large Redrock Mines"
+ building_redrock_mines_03_desc:1 "$building_rammelsberg_mines_03_desc$"
+ building_redrock_mines_04:1 "Redrock Mining Complex"
+ building_redrock_mines_04_desc:1 "$building_rammelsberg_mines_04_desc$"
+ 
+ building_obelisk_of_the_sun_01:1 "Obelisk of the Sun"
+ building_obelisk_of_the_sun_01_desc:1 "One of three obelisks in Uldum that were once used to mask the area from the outside world."
+ building_obelisk_of_the_moon_01:1 "Obelisk of the Moon"
+ building_obelisk_of_the_moon_01_desc:1 "One of three obelisks in Uldum that were once used to mask the area from the outside world."
+ building_obelisk_of_the_stars_01:1 "Obelisk of the Stars"
+ building_obelisk_of_the_stars_01_desc:1 "One of three obelisks in Uldum that were once used to mask the area from the outside world."
```

Comment:
Could make one line for "building_obelisks_desc" having the actual text in and then using that in all 3 descriptions. But fine this way, too.